src/effects/EffectUI.h

```diff
@@ -96,7 +95,7 @@ class EffectUIHost final : public wxDialogWrapper
 
    AudacityProject &mProject;
    wxWindow *mParent;
-   Effect &mEffect;
+   EffectUIHostInterface &mEffect;
```

Comment:
It would help me read and understand this code some weeks from now, if we could rename `mEffect` to `mEffectUIHost` or `mEffectUIHostInterface`